### PR TITLE
Import of Docker recipe for container

### PR DESCRIPTION
**Pull request by willend**
Here is the container recipe as it currently looks.

"Fat" container with JupyterLab, X/vnc Desktop, McStas, Python bells and whistles.

(Corresponding to the build docker.io/mccode/dmsc-summerschool:1.3)

### CODE REVIEW

**Review by jl-wynen — COMMENTED** [review]
Why are there so many files? Did you commit the image itself?

File: container/LICENSE [hunk 1]
```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2019, University of Dundee
```
[the remark this answers is not in the document]
Comment (willend) [reply container/LICENSE:3]: But hey, if you prefer me not volunteering to help out, just let me know.

[the remark this answers is not in the document]
Comment (jl-wynen) [reply container/LICENSE:3]: That's not what I'm saying! I was just confused what this is. I have never seen most of these files.

[the remark this answers is not in the document]
Comment (willend) [reply container/LICENSE:3]: OK, fair enough.

**Comment by nvaytet** [comment]
I think I also agree that all this seems overkill?
I understand that you have an old container that worked and you just added new things to it.
But since we only need the jupyter hub and a conda environment, and not all the desktop env and VNC stuff, I would vote to have this cleaned up before adding here?

I am not saying we cannot use it for the school, but there are some VNC binaries, icons, etc... that I don't think this repo is the best place to store?

**Comment by willend** [comment]
No worries, I keep the actual container code elsewhere then.

**Pull request closed without merging**